private/extensions/maven.bzl

```diff
@@ -267,9 +273,14 @@ def _maven_impl(mctx):
 
             repo = repos.get(install.name, {})
 
+            repo["resolver"] = install.resolver
```

Comment:
This is errorprone if the there's a non-root module that contributes to the same '@maven' repo. This was what I was confused by when trying this out on the Bazel repo: `protobuf`'s `maven.install` contributes `resolver = 'coursier'`, which overrode my root's `resolver = 'maven'`.

Reply:
I've tweaked the way this works. Now, if any of the below attributes are set in the root module, they override the values from other tags.

  * `fail_if_repin_required`
  * `fail_on_missing_checksum`
  * `fetch_javadoc`
  * `fetch_sources`
  * `resolver`
  * `strict_visibility`

I suspect I need to do this with `repositories` too.